master/setup.py

```diff
@@ -391,15 +391,9 @@ def define_plugin_entries(groups):
         'twisted >= 11.0.0',
         'Jinja2 >= 2.1',
         'zope.interface >= 4.1.1',  # required for tests, but Twisted requires this anyway
-        'future'
-    ]
-
-    setup_args['install_requires'] += [
-        # sqlalchemy-0.8 betas show issues with sqlalchemy-0.7.2, so stick to 0.7.10
-        'sqlalchemy >= 0.6, <= 0.7.10',
-        # buildbot depends on sqlalchemy internals, and this is the tested
-        # version.
-        'sqlalchemy-migrate==0.7.2',
+        'future',
+        'sqlalchemy==0.7.10, >0.8',
```

Comment:
I think we discussed this kind of things before: the new pip interprets this as '==0.7.10 && > 0.8'...  no?